### PR TITLE
Add gRPC server for starting jobs

### PR DESCRIPTION
**Pull request by JKoetsier**
gRPC server. Accepts job start messages now.

Generated files (in the out and protos directories) are included in the repo. No auto-regeneration is done at the moment. Proto can be used to generate the JS stubs.

gRPC server is not behind Gatekeeper yet.

### CODE REVIEW

File: src/gobmanagement/grpc/codegen.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+    def _collect_commands(self):
+        """Populates self.messages and self.services based on the start commands definition.
+
+        :return:
+        """
+        commands = StartCommands()
+        response_message = 'JobResponse'
+
+        for _, command in commands.get_all().items():
```
Comment (jasper-jager) [inline src/gobmanagement/grpc/codegen.py]: .values() can be used

File: src/gobmanagement/grpc/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+directory is generated from the StartCommands defined in GOB Core.
+From that, the Python skeleton code in ```out``` is generated using the
+gRPC proto compiler.
+
+The ```services``` directory contains our implementation.
+
+## Running codegen.py
+```codegen.py``` only needs to be run after the a change is made in the
+start commands.
+
```
Comment (Vegter) [inline src/gobmanagement/grpc/README.md]: Why not make this part of the build process?

**Comment by Vegter** [comment]
FAIL Required test coverage of 80% not reached. Total coverage: 78.99%

That's a minimal difference...

**Comment by Vegter** [comment]
General remark: Nice work !

File: src/gobmanagement/grpc/codegen.py (hunk 1 above)
Comment (JKoetsier) [reply src/gobmanagement/grpc/codegen.py]: Correct :) Changed.

File: src/gobmanagement/grpc/README.md (hunk 2 above)
Comment (JKoetsier) [reply src/gobmanagement/grpc/README.md]: Included now :)

**Pull request merged**